### PR TITLE
docs: epic dotfiles plug

### LINKED ISSUE #8: How do i set this exactly like the screenshots?
These are just the colors... and they aren't even working, how do i even use @COLOR?

### PR DESCRIPTION
**Pull request by rubyowo**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@
 
 -	Q: **_"Waybar doesn't work with the colors"_**\
 	A: Make sure you included the file in the right place and you are using `@COLOR`
+- Q: **_Hey! Can I borrow that bar config?_**\
```
Comment (sgoudham) [inline README.md]:
```suggestion
- Q: **_How can I make the bar look like the preview image?_**\
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -60,6 +60,8 @@
 
 -	Q: **_"Waybar doesn't work with the colors"_**\
 	A: Make sure you included the file in the right place and you are using `@COLOR`
+- Q: **_How can I make the bar look like the preview image?_**\
+  A: Sure, here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) in the preview image.
```
Comment (sgoudham) [inline README.md]:
```suggestion
  A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) for the preview image.
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -60,6 +60,8 @@
 
 -	Q: **_"Waybar doesn't work with the colors"_**\
 	A: Make sure you included the file in the right place and you are using `@COLOR`
+- Q: **_How can I make the bar look like the preview image?_**\
```
Comment (sgoudham) [inline README.md]:
```suggestion
- Q: **_"How can I make the bar look like the preview image?"_**\
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

**Comment by nullishamy** [comment]
# Lets Get This Merged

File: README.md [hunk 4]
```diff
@@ -60,6 +60,8 @@
 
 -	Q: **_"Waybar doesn't work with the colors"_**\
 	A: Make sure you included the file in the right place and you are using `@COLOR`
+- Q: **_"How can I make the bar look like the preview image?"_**\
+  A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) for the preview image.
```
Comment (sgoudham) [inline README.md]:
```suggestion
  A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/f925cf8e3461420a21b6dc8b8ad1190107b0cc56/config/waybar) for the preview image.
```

**Review by sgoudham — CHANGES_REQUESTED** [review]

**Comment by rubyowo** [comment]
my pr has been ruined by catppuccin scuff #cancelcatppuccin

**Review by sgoudham — APPROVED** [review]

**Review by nullishamy — APPROVED** [review]
lfg

**Pull request merged**